lesson_21/script.js

```diff
@@ -0,0 +1,23 @@
+let numberAdd1 = prompt('insert first number for addition');
+let numberAdd2 = prompt('insert second number for addition');
+
+let sumAdd = (+numberAdd1) + (+numberAdd2);
+alert('the addition of numeric ' +numberAdd1+ ' and numeric ' +numberAdd2+ ' give us ' +sumAdd );
+
+let numberSubstr1 = prompt('insert first number for substraction');
+let numberSubstr2 = prompt('insert second number for substraction');
+
+let sumSubstr = (+numberSubstr1) - (+numberSubstr2);
```

Comment:
these all operations could be done on the same numbers what user entered

that's okay to have it separated but in the future HomeTask if it will be easier for you to have the same values, that's completely fine!